ui/src/main/js/util/httpErrorUtilities.js

```diff
@@ -15,11 +16,13 @@ export function createErrorObject(errorResponse) {
         return createEmptyErrorObject();
     }
     const responseMessage = errorResponse.message;
+    const { isDetailed } = errorResponse;
     const responseErrors = errorResponse.errors;
```

Comment:
Why not destructure all of these and then just map them to the correct name in the return?

Reply:
message is defined as a const below on line 19 so we cant' destructure without a variable conflict.  In order to destructure but we would have to use a different variable name and syntax to set the message property in the response object.

`'message': responseMessage`